content/blog/kubernetes-clusters-as-code-part1.md

```diff
@@ -0,0 +1,1094 @@
+hpegl_caas_cluster.test: Still creating... [22m0s elapsed]
+hpegl_caas_cluster.test: Still creating... [22m10s elapsed]
+hpegl_caas_cluster.test: Still creating... [22m20s elapsed]
+hpegl_caas_cluster.test: Still creating... [22m30s elapsed]
+hpegl_caas_cluster.test: Still creating... [22m40s elapsed]
+hpegl_caas_cluster.test: Still creating... [22m50s elapsed]
+hpegl_caas_cluster.test: Still creating... [23m0s elapsed]
+hpegl_caas_cluster.test: Still creating... [23m10s elapsed]
+hpegl_caas_cluster.test: Still creating... [23m20s elapsed]
+hpegl_caas_cluster.test: Still creating... [23m30s elapsed]
+hpegl_caas_cluster.test: Still creating... [23m40s elapsed]
+hpegl_caas_cluster.test: Still creating... [23m50s elapsed]
+hpegl_caas_cluster.test: Still creating... [24m0s elapsed]
+hpegl_caas_cluster.test: Still creating... [24m10s elapsed]
+hpegl_caas_cluster.test: Still creating... [24m20s elapsed]
+hpegl_caas_cluster.test: Still creating... [24m30s elapsed]
+hpegl_caas_cluster.test: Still creating... [24m40s elapsed]
+hpegl_caas_cluster.test: Still creating... [24m50s elapsed]
+hpegl_caas_cluster.test: Still creating... [25m0s elapsed]
+hpegl_caas_cluster.test: Still creating... [25m10s elapsed]
+hpegl_caas_cluster.test: Still creating... [25m20s elapsed]
+hpegl_caas_cluster.test: Still creating... [25m30s elapsed]
+hpegl_caas_cluster.test: Still creating... [25m40s elapsed]
+hpegl_caas_cluster.test: Still creating... [25m50s elapsed]
+hpegl_caas_cluster.test: Still creating... [26m0s elapsed]
+hpegl_caas_cluster.test: Still creating... [26m11s elapsed]
+hpegl_caas_cluster.test: Still creating... [26m21s elapsed]
+hpegl_caas_cluster.test: Still creating... [26m31s elapsed]
+hpegl_caas_cluster.test: Still creating... [26m41s elapsed]
+hpegl_caas_cluster.test: Still creating... [26m51s elapsed]
+hpegl_caas_cluster.test: Still creating... [27m1s elapsed]
+hpegl_caas_cluster.test: Still creating... [27m11s elapsed]
+hpegl_caas_cluster.test: Still creating... [27m21s elapsed]
+hpegl_caas_cluster.test: Still creating... [27m31s elapsed]
+hpegl_caas_cluster.test: Still creating... [27m41s elapsed]
+hpegl_caas_cluster.test: Still creating... [27m51s elapsed]
+hpegl_caas_cluster.test: Still creating... [28m1s elapsed]
+hpegl_caas_cluster.test: Still creating... [28m11s elapsed]
+hpegl_caas_cluster.test: Still creating... [28m21s elapsed]
+hpegl_caas_cluster.test: Still creating... [28m31s elapsed]
+hpegl_caas_cluster.test: Still creating... [28m41s elapsed]
+hpegl_caas_cluster.test: Still creating... [28m51s elapsed]
+hpegl_caas_cluster.test: Still creating... [29m1s elapsed]
+hpegl_caas_cluster.test: Creation complete after 29m1s [id=8a3396db-ae26-44fd-a128-264c357f71fb]
+ 
+Apply complete! Resources: 1 added, 0 changed, 0 destroyed.
+```
+
+<!--StartFragment-->
+
+Based on the cluster details provided, the cluster will get created and the same can be verified in the HPE GreenLake Central, under the list of clusters:
```

Comment:
Don't you need to launch the CaaS widget from HP EGreenLake central console to visualize/verify the K8s cluster has been successfully created?

Reply:
Yes, I have added this step.